sources/packages/web/src/views/aest/student/Profile.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <tab-container>
-    <student-profile :studentId="studentId" />
+    <student-profile
+      :studentId="studentId"
+      :allowDisabilityStatusUpdate="true"
```

Comment:
Should this not be defined based on the roles of the Ministry?

Reply:
I checked other similar models, and saw that we were using the ministry roles to restrict at only at the primary button of the model and not any earlier in the flow. If I use the ministry role here, it will hide the button to open the modal to all not authorized ministry users unlike the behavior of other models.